code/datums/components/slippery.dm

```diff
@@ -20,12 +20,12 @@
 	var/slip_tiles
 	/// TRUE If this slip can be avoided by walking.
 	var/walking_is_safe
-	/// TRUE if having no slip shoes makes you immune to this slip.
-	var/noslip_is_immune
+	/// FALSE if you want no slip shoes to make you immune to the slip
+	var/slip_always
```

Comment:
slip_always reads a bit odd. `always_slip` perhaps?

Reply:
the carbon slip arg is `SlipAny`. I'm doing this for consistency.